src/sentry/api/endpoints/organization_issue_breakdown.py

```diff
@@ -0,0 +1,137 @@
+from datetime import datetime, timedelta
+from typing import TypedDict
+
+from django.db.models import Count, F, Q
+from django.db.models.functions import TruncDay
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import EnvironmentMixin, region_silo_endpoint
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.helpers.environments import get_environments
+from sentry.api.utils import get_date_range_from_params
+from sentry.models.group import Group, GroupCategory, GroupStatus
+from sentry.models.organization import Organization
+from sentry.models.project import Project
+
+CATEGORY_MAP = {
```

Comment:
what about performance issues and other issue categories?

Reply:
Not wanted right now AFAIK.  We can certainly add those.  @ryan953 could provide more details on the project's goals.

Reply:
hmm yeah seems strange to include feedbacks but not the other issue types. curious to see about that!

Reply:
Lets sum up performance, error and other issue types into one number. We're breaking these down by resolution value, not by type.
I don't think it makes sense to break down by type, because that's largely an internal thing because of our two data models. Some categories like Replay Issue also have low volumes and would just make a chart legend messy.

Feedbacks should stay broken out imo. If you want them then need to make an independant api query. My reasoning for making these special is that: they arrive at wildly different volumes, have no fingerprinting, and the resolution workflow is a bit different including the fact that a feedback might have an error attached to it (so we'd be double counting).